README.md

```diff
@@ -38,6 +38,16 @@ class ExpringJob < ActiveJob::Base
 end
 ```
 
+Allow to subscribe on worker perform method and detect errors
+
+```ruby
+ActiveSupport::Notifications.subscribe('fleiss.worker.perform') do |event|
```

Comment:
looking at e.g. `grape` for reference this should be `worker_perform.fleiss`, it's also e.g. `render_template.action_view`, etc